Remove FAQ and fix links (#86)
ezsystems/user-documentation: docs/personalization/content_import.md

!!! tip

>     You can find Specification for Import interfaces in the developer guide under [Content API](https://doc.ibexa.co/en/master/guide/personalization/developer_guide/content_api).

Reviewer: ```suggestion
    For a specification for import interfaces, see [Content API](https://doc.ibexa.co/en/master/guide/personalization/developer_guide/content_api)  in developer documentation.
```
